EIPS/eip-draft_createlink.md

```diff
@@ -0,0 +1,74 @@
+---
+eip: <to be assigned>
+title: CREATELINK opcode
+author: William Morriss (@wjmelements)
+discussions-to: [Github PR](https://github.com/ethereum/EIPs/pull/2185)
+status: Draft
+type: Standards Track
+category Core
+created: 2019-07-09
+replaces: 911
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+A cheaper way to proliferate identical smart contracts.
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+This EIP specifies a new opcode, `CREATELINK`, which creates a new contract matching the code of an existing address.
+
+## Motivation
+<!--The motivation is critical for EIPs that want to change the Ethereum protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the EIP solves. EIP submissions without sufficient motivation may be rejected outright.-->
+Hundreds of thousands of identical contracts are proliferating from contract factories.
+Because they share the same immutable code, a node implementation could detect these identical contracts and share their storage similar to how hard-linked files share theirs.
+New links require less storage than new code.
+Therefore, empowering this design with an opcode will reduce the overhead of operating a node long-term.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+The opcode for `CREATELINK` is `0xf7`.
+`CREATELINK` pops one word, the target address.
+As with `BALANCE`, `EXTCODESIZE`, `EXTCODECOPY`, and `EXTCODEHASH`, the upper 12 bytes are zeroed.
```

Comment:
The first time any opcode is referenced in an EIP it needs to have (\`0x##\`) after it.  Please add that for these opcodes.

Alternatively, just remove them since we don't really need a comparison in order to draft a specification.  Instead just say:
> the upper 12 bytes of the popped address are zeroed.